mantidproject/mantid
# Indirect Simulations SimulatedDensityOfStates - Allow algorithm to parse isotopes

The SimulatedDensityOfStates algorithm should now be able to parse isotopes in the format `element:isotope` correctly. A new test case for this has been added to the unit test and a new test file has also been added. 

**To test:**
- Download the new test file from by building the unit test data
- Run the following script:

```
isotope_phonon = 'test_isotopes.phonon'
wks_group = SimulatedDensityOfStates(PHONONFile=isotope_phonon, SpectrumType='DOS', Ions='H:P,C:P,O')
```
- Ensure that the output workspace group contains three workspaces named `H(2)`, `C(13)` and `O`
- Each workspace should have 2 spectra (Workspace index 0 = `Guassian`, Workspace index 1 = `Stick`)
- Check the sample materials on each workspace to ensure the sample material has been correctly set to the isotope of the element _you can use the [NIST table](https://www.ncnr.nist.gov/resources/n-lengths/list.html) to ensure scatter cross section are correct_

Fixes #16981

[RELEASE NOTES](https://github.com/mantidproject/mantid/blob/40c0e9c6f35db398a25a2566d2cf7fd7d1f868c4/docs/source/release/v3.8.0/indirect_inelastic.rst)

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
